Keter/Types/V10.hs

```diff
@@ -134,6 +139,7 @@ instance Default KeterConfig where
         , kconfigExternalHttpPort = 80
         , kconfigExternalHttpsPort = 443
         , kconfigEnvironment = Map.empty
+        , kconfigConnectionTimeBound = 5000
```

Comment:
Ooops! Excuse me! Also `etc/keter-config.yaml` need to be updated (same value there).